src/components/tour/tour_step.tsx

```diff
@@ -131,6 +149,26 @@ export const EuiTourStep: FunctionComponent<EuiTourStepProps> = ({
     );
   }
 
+  const [willMount, setWillMount] = useState<boolean>(false);
```

Comment:
This state name was pretty confusing to me at first because I was thinking you maybe wanted something like `useUpdateEffect`. Can I suggest renaming it to something more specific to anchors like:

```suggestion
  const [hasValidAnchor, setHasValidAnchor] = useState<boolean>(false);
```

Thoughts?